Changelog.md

```diff
@@ -23,6 +23,7 @@ Bugfixes:
  * SMTChecker: Fix pointer knowledge erasing in loops.
  * SMTChecker: Fix internal error when using compound bitwise assignment operators inside branches.
  * SMTChecker: Fix internal error when inlining a function that returns a tuple containing an unsupported type inside a branch.
+ * SMTChecker: Fix internal error when inlining functions from a different source that use state variables.
```

Comment:
```suggestion
 * SMTChecker: Fix internal error when inlining functions from a different source that uses state variables.
```

Reply:
I think the funcitons use the state variables, not the source.

Reply:
Ah. I got confused then.